Automattic/wp-calypso
# ETK Timeline Block: Use enableAlpha=true to allow 'transparent' as an available `background` color

Fixes #63626

Building on evaluation in https://github.com/Automattic/wp-calypso/pull/65104#issuecomment-1170735113

Opened https://github.com/WordPress/gutenberg/issues/42170 to see if there's a better way.

## Proposed Changes

Uses `enableAlpha={ true }` to make transparency available to the Timeline Block:

<img width="621" alt="image" src="https://user-images.githubusercontent.com/36432/179060908-ec040346-71e6-4032-a3b1-4054fcc1ca9c.png">


This lets the user override the default `background` color, and let the background color of the page appear.

#### Testing Instructions

1. Sandbox a WordPress.com site.
2. Install ETK from branch: `install-plugin.sh editing-toolkit improve/timeline-block-transparent-color`
3. Create a new post.
4. Add a "Timeline Block" to the post.
5. Verify the Timeline Block has the default background color `#EEEEEE`:

<img width="1597" alt="image" src="https://user-images.githubusercontent.com/36432/177417894-b706ea85-16d6-4ec6-a451-7c523e619ed0.png">

<img width="1199" alt="image" src="https://user-images.githubusercontent.com/36432/177418008-eae81599-d67e-4cf0-ab15-f5c54fa66955.png">

6. Verify the Timeline Block allows accessing the alpha channel of the color picker:

<img width="621" alt="image" src="https://user-images.githubusercontent.com/36432/179060908-ec040346-71e6-4032-a3b1-4054fcc1ca9c.png">

7. Select a transparent color and verify it's applied to the Timeline Block:

<img width="1518" alt="image" src="https://user-images.githubusercontent.com/36432/179061198-904cf8c9-6b7b-4e05-9ea9-a328493d6442.png">

<img width="1088" alt="image" src="https://user-images.githubusercontent.com/36432/179061297-1e588406-bba2-4059-8c12-cdb1502b19c3.png">

8. Switch to the Code Editor, and then back to the Visual Editor and verify the Timeline Block isn't crashed.
